haskell-src/Concordium/Types/Queries/Tokens.hs

```diff
@@ -39,3 +68,82 @@ data TokenAccountState = TokenAccountState
       memberDenyList :: !Bool
     }
     deriving (Eq, Show)
+
+-- | Parameter for a Token module.
+newtype TokenParameter = TokenParameter {parameterBytes :: BS.ShortByteString}
+    deriving (Eq)
+    deriving (Aeson.ToJSON, Aeson.FromJSON, Show) via ByteStringHelpers.ByteStringHex
```

Comment:
I'm seeing this the first time for the derivation of `ToJSON`. Are we always using `ByteStringHelpers.ByteStringHex` for the derivation of JSON instances for byte strings?

Reply:
Can't say whether we always do this, but we do for smart contract parameters, contract events, memos, register data at least, which are all arbitrary bytestrings.